bin/storm.py

```diff
@@ -66,6 +66,12 @@ def init_storm_env():
     for option in options:
         value = config.get('environment', option)
         os.environ[option] = value
+        
+def get_java_cmd():
```

Comment:
The one liner should have done the same thing, except when JAVA_HOME wasn't set. But in that case it's okay to just call "java", since you don't have to postfix with ".exe" for binaries on the PATH in Windows. I agree that this is better though :)